codec_packetizers/g711/g711_depacketizer.c

```diff
@@ -0,0 +1,127 @@
+/* Standard includes. */
+#include <string.h>
+
+/* API includes. */
+#include "g711_depacketizer.h"
+
+G711Result_t G711Depacketizer_Init( G711DepacketizerContext_t * pCtx,
+                                    G711Packet_t * pPacketsArray,
+                                    size_t packetsArrayLength )
+{
+    G711Result_t result = G711_RESULT_OK;
```

Comment:
May init with result not ok? And mark it good when passing checks?
Just an idea - may a bad one - just think of it....

Reply:
All right - keep it consistent.